bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/thing/station.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sncf"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="station">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="api"/>
+		</supported-bridge-type-refs>
+
+		<label>Station</label>
+
+		<channel-groups>
+			<channel-group id="arrivals" typeId="passage">
+				<label>Next Arrival</label>
+			</channel-group>
+			<channel-group id="departures" typeId="passage">
+				<label>Next Departure</label>
+			</channel-group>
+		</channel-groups>
+
+		<representation-property>stopPointId</representation-property>
+
+		<config-description>
+			<parameter name="stopPointId" type="text" required="true">
+				<label>Station ID</label>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-group-type id="passage">
+		<label>Other</label>
+		<channels>
+			<channel id="direction" typeId="direction"/>
+			<channel id="code" typeId="code"/>
+			<channel id="commercialMode" typeId="commercialMode"/>
+			<channel id="name" typeId="name"/>
+			<channel id="network" typeId="network"/>
+			<channel id="timestamp" typeId="timestamp"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="direction">
+		<item-type>String</item-type>
+		<label>Direction</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="code">
+		<item-type>String</item-type>
+		<label>Code</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="name">
+		<item-type>String</item-type>
+		<label>Name</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="network">
+		<item-type>String</item-type>
+		<label>Network</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="commercialMode">
+		<item-type>String</item-type>
+		<label>Commercial Mode</label>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="timestamp">
+		<item-type>DateTime</item-type>
+		<label>Timestamp</label>
+		<state readOnly="true"/>
```

Comment:
Please add a default pattern with only hours and minutes.

Reply:
Done

Reply:
You followed my comment, thank you.
But now I see there is one mine between the arrival and the departure time with the default pattern, like 19:16 and 19h17. With the full pattern, I remember the difference was 40 seconds and so the departure time was in the same minute.
All that to say that maybe you should include seconds in the default pattern ?

Reply:
Sometimes I have the same minute with the current code, so first I should check the raw data.

Reply:
I think it will be clearer  if I reintroduce seconds in the pattern.